vm/actor/src/builtin/verifreg/types.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use crate::StoragePower;
+use address::Address;
+use num_bigint::biguint_ser::{BigUintDe, BigUintSer};
+
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use std::{
+    fmt,
+    ops::{Add, Sub},
+};
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct VerifierParams {
+    pub address: Address,
+    pub allowance: Datacap,
+}
+
+pub type AddVerifierParams = VerifierParams;
+pub type AddVerifierClientParams = VerifierParams;
+
+pub const MINIMUM_VERIFIED_SIZE: u32 = 1 << 20;
+
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub struct Datacap(StoragePower);
```

Comment:
Why is this not just an alias? There is no functionality added on top of this type that you don't get already, and you miss out on the BigUint API